Training2/src/task3/src/App.js

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+import "./App.css";
+
+function App() {
+  const [value, setValue] = useState("");
+  const [valueCountDown, setValueCountDown] = useState(0);
+  const [isStop, setIsStop] = useState(false);
+  const handleChange = (e) => {
+    setValue(e.target.value);
+  };
+
+  useEffect(() => {
+      const countdown = setTimeout(() => {
+        if (valueCountDown>0 && isStop===false)
+        setValueCountDown(valueCountDown - 1);
+      }, 1000);
+      return ()=>clearTimeout(countdown);
+  }, [valueCountDown, isStop]);
+
+  function handleCountdown(number) {
+    console.log(number);
+    setIsStop(false);
```

Comment:
@phuongluongZigvy 
when you call like that, in the useEffect it can run with isStop is false, but countdown value is not equal `number` you can group these fields in the object state, and setState one time